src/vivarium_gates_iv_iron/utilities.py

```diff
@@ -102,4 +102,18 @@ def get_norm_from_quantiles(mean: float, lower: float, upper: float,
                             quantiles: Tuple[float, float] = (0.025, 0.975)) -> stats.norm:
     stdnorm_quantiles = stats.norm.ppf(quantiles)
     sd = (upper - lower) / (stdnorm_quantiles[1] - stdnorm_quantiles[0])
-    return stats.norm(loc=mean, scale=sd)
\ No newline at end of file
+    return stats.norm(loc=mean, scale=sd)
+
+
+def get_truncnorm_from_quantiles(mean: float, lower: float, upper: float,
+                                 quantiles: Tuple[float, float] = (0.025, 0.975),
+                                 lower_clip: float = 0.0, upper_clip: float = 1.0) -> stats.truncnorm:
+    stdnorm_quantiles = stats.norm.ppf(quantiles)
+    sd = (upper - lower) / (stdnorm_quantiles[1] - stdnorm_quantiles[0])
+    try:
```

Comment:
This is fine, but I think I'd prefer just to check if `sd == 0` rather than catching the divide by 0 error.

Reply:
I considered that, but why check for a rare case when you can try/except and avoid the branch in the normal case?